xknxproject/const.py

```diff
@@ -4,6 +4,7 @@
 ETS_6_SCHEMA_VERSION: Final = 21
 ETS_5_7_SCHEMA_VERSION: Final = 20
 ETS_5_6_SCHEMA_VERSION: Final = 14
+ETS_4_SCHEMA_VERSION: Final = 11
```

Comment:
```suggestion
ETS_4_2_SCHEMA_VERSION: Final = 11
```
4.0 seems to have used version 10

Reply:
Yep, will change that.

Reply:
Done. Sorry, I could have applied your suggestion.

Reply:
Doesn't make a difference 🤷
We should change the ETS 6 thing to `ETS_6_0_SCHEMA_VERSION` too, but that can be done in another PR too... 5_6 also doesn't seem correct, but not sure if that is even used anywhere